.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
There are more non-default options in https://www.npmjs.com/package/@ianvs/prettier-plugin-sort-imports#usage, but this seems to be the minimal to get the plugin to do something. I'm totally unopinionated on the actual style we use, so happy to configure anything anybody wants here. I just want automation, so I can stop thinking about it.